reports/methodology.md

```diff
@@ -20,13 +20,19 @@ Alongside these features include:
 - author (Reddit user name)
 - date
 - post
-## Data cleaning
+## Data cleaning/transformation
+
+In order to answer the question "How has the substance use increased over the pandemic?", the feature `substance_use_total` is selected as the target feature, and the data is cleaned to focus exclusively on this feature.
+
+### Data Cleaning Steps
+- Combine the `pre` and `post` datasets into one dataset. 
+- Filter the dataset to keep only the columns of interest, including `subreddit`, `author`, `date`, `post`, and `substance_use_total`.
+### Feature Engineering
+- We apply a simple method by adding a new feature `period` to indicate the timeframe of the posts (before or after the pandemic). This helps to represent the data better and to compare the timeframes more easily in later parts of this report. 
+### Room for improvement
+- It is noted that in our `process_raw.py` script, we used the `try-except` block to make the script runnable. Although we are sure that this `try-except` block does not affect our raw data processing, it is suggested that it should not be used in other circustances. 
```

Comment:
Also, i don't think we want to say that try-except _shouldn't_ be used in other circumstances.
Try-catches are necessary for error handling which is always useful to have and should be included when writing new functionality. 

I think in this case though, we're using the try-catch in an unconventional way.. you can maybe describe this situation (why we need the try-except) and just say it could be improved if we had more time.

Reply:
I understand. I will revise this part accordingly.